iTerm/fzf.md

```diff
@@ -41,7 +41,7 @@ fd() {
 }
 ```
 
-```sh
+```console
```

Comment:
I would like to keep this as console since this is a pre install command. Let me know if you have a differing opinion.. will change it. For now i would like to maintain this idea.

Reply:
```sh
# fh - search in your command history and execute selected command
fh() {
  eval $( ([ -n "$ZSH_NAME" ] && fc -l 1 || history) | fzf +s --tac | sed 's/ *[0-9]* *//')
}
```
```console
# fh - search in your command history and execute selected command
fh() {
  eval $( ([ -n "$ZSH_NAME" ] && fc -l 1 || history) | fzf +s --tac | sed 's/ *[0-9]* *//')
}
```